1223-curator_worklist/my_worklist_approval.feature

```diff
@@ -22,15 +22,15 @@ Feature: Curator handles an Approval Request
 		And the User has created an Approval Request
 		And the Curator views the landing page of the Resource assosiated with the Request
 
-	@needJiraTag
+	@needJiraTagApproval
 	Scenario: Curator approves an Approval Request to publish
 		Given the User wants to publish a Resource
 		And the Editor has restricted publishing rights to Curators
 		When the Curator approves the Request
 		Then the Request status is changed to "Approved"
 		And the Resource is published
 
-	@needJiraTag
+	@needJiraTagApproval
 	Scenario: Curator approves an Approval Request to grant open access to content
 		Given the User has published a Resource
 		And the Editor demands a Approval to open access to Resources' content
```

Comment:
Det er egentlig Customer Institution som krever Approval, heller enn Editor? Men det er muligens mer en teknikalitet å regne

Reply:
Editor on behalf of institution…?